packages/api-plugin-fulfillment/src/resolvers/index.js

```diff
@@ -0,0 +1,19 @@
+import FulfillmentMethod from "./FulfillmentMethod/index.js";
+import Mutation from "./Mutation/index.js";
+import Query from "./Query/index.js";
+
+/**
+ * Fulfillment related GraphQL resolvers
+ * @namespace Fulfillment/GraphQL
+ */
```

Comment:
Is there a reason this comment is just hanging out here by itself?

Reply:
:-) 
No, it was copied over from api-shipments. Removed it.